cnd/src/http_api/routes/index.rs

```diff
@@ -83,16 +84,24 @@ pub async fn post_han_ethereum_halight_bitcoin(
         .map_err(problem::from_anyhow)
         .map_err(warp::reject::custom)?;
 
-    let reply = warp::reply::reply();
+    let swap_params: HanEtherereumHalightBitcoinCreateSwapParams = body.clone().into();
 
-    let id = LocalSwapId::default();
+    let swap_id = LocalSwapId::default();
+    let reply = warp::reply::reply();
 
-    facade.save(id, ()).await;
+    let swap: CreatedSwap<HanEthereumEther, HalightLightningBitcoin> = CreatedSwap {
```

Comment:
```suggestion
    let swap = CreatedSwap::<HanEthereumEther, HalightLightningBitcoin> { {
```

Reply:
In the name of learning, is there any reason why you favour the turbo fish syntax over the explicit type syntax @D4nte?

Reply:
I usually prefer turbo-fish because it some robust.

Annotating the final type breaks if you refactor the code by chaining more things onto the expression for example.
Turbo-fish syntax doesn't suffer from that because it is localized to the actual type that needs hints.

Reply:
It's simply shorter @tcharding. Your code repeat the `CreateSwap` symbol twice, the information for the reader is redundant.

Reply:
Please note if you mark a comment as "resolved" it does not entice the author to reply to the comment :)